comps/comps-foreman-rhel7.xml

```diff
@@ -33,6 +33,7 @@
       <packagereq type="default">foreman-service</packagereq>
       <packagereq type="default">foreman-telemetry</packagereq>
       <packagereq type="default">foreman-vmware</packagereq>
+      <packagereq type="default">nodejs-graphql-tag</packagereq>
```

Comment:
Can you please include https://github.com/theforeman/foreman-packaging/pull/6772 here too?

Reply:
Added.